projects/plugins/mu-wpcom-plugin/changelog/add-launchpad-is-enabled-callback

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: added
+
+Fix mu-wpcom-plugin lint error
```

Comment:
Can you separate this fix out into a fresh PR? This one is already big and it's confusing to have two changelogs for one PR.